This is completely unrelated so feel free to table it, but starting today, the test for [`cli/api/http_client_test.go:225`](https://github.com/cli/cli/blob/590208f5d6760b427f7120dcc7c14a6310947096/api/http_client_test.go#L225) is failing locally.
```
--- FAIL: TestHTTPClientSanitizeJSONControlCharactersC0 (0.00s)
    http_client_test.go:225: 
                Error Trace:    /[redacted]/cli/api/http_client_test.go:225
                Error:          Not equal: 
                                expected: "1^A 2^B 3^C 4^D 5^E 6^F 7^G 8^H 9\t A\r\n B\v C^L D\r\n E^N F^O"
                                actual  : "1^A 2^B 3^C 4^D 5^E 6^F 7^G 8\b 9\t A\r\n B\v C\f D\r\n E^N F^O"
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,3 +1,3 @@
                                -1^A 2^B 3^C 4^D 5^E 6^F 7^G 8^H 9       A
                                - B
                                    C^L D
                                +1^A 2^B 3^C 4^D 5^E 6^F 7^G  9  A
                                + B
                                    C
                                      D
                                  E^N F^O
                Test:           TestHTTPClientSanitizeJSONControlCharactersC0
FAIL
```

I'm sure I'm missing something, but my `actual` output seems correct to me:
```
8\u0008 -> 8\b (backspace)
```
```
C\u000c -> C\f (form feed)
```
However, the tests expect `8^H` and `C^L` respectively. Would you know why those are the expected values? I'm not sure if this is a bug or if I'm misunderstanding something.

--- a/pkg/cmd/project/view/view.go
+++ b/pkg/cmd/project/view/view.go
@@ -94,10 +94,7 @@ func runView(config viewConfig) error {
 	}
 
 	if config.opts.web {
-		if err := config.URLOpener(project.URL); err != nil {
-			return err
-		}
-		return nil
+		return config.URLOpener(project.URL)
 	}
 
 	if config.opts.exporter != nil {

